installer/templates/new/assets/javascripts/widget.js

```diff
@@ -23,6 +23,17 @@ class Widget extends React.Component {
           setTimeout((() => window.location.reload()), 5 * 60 * 1000)
         }
       });
+
+      this._events.addEventListener('_kitto', (event) => {
```

Comment:
Not caching the internal Kitto event in the first place seems a perfectly sane option.